packages/node_modules/@ciscospark/react-container-activity-list/src/selectors.js

```diff
@@ -75,20 +75,28 @@ export const getActivityList = createSelector(
         const isAdditional = sameDay && lastActorId === activity.actor.id && lastVerb === activity.verb;
         lastActorId = activity.actor.id;
         lastVerb = activity.verb;
+
         // eslint-disable-reason callbacks are necessary
         // eslint-disable-next-line max-nested-callbacks
         const isFlagged = flags && flags.some((flag) => flag.activityUrl === activity.url);
+
+        // Update verbs are currently used for call data
+        let callData;
+        if (activity.verb === `update`) {
+          callData = getCallData(activity, currentUser);
+        }
+
         const visibleActivity = {
           type: ITEM_TYPE_ACTIVITY,
-          activity,
+          activity: {
+            ...activity,
+            callData
```

Comment:
`callData` is added to the activity object so we don't clutter up the top level params.